lib/src/install.rs

```diff
@@ -946,7 +946,7 @@ pub(crate) fn reexecute_self_for_selinux_if_needed(
     override_disable_selinux: bool,
 ) -> Result<SELinuxFinalState> {
     // If the target state has SELinux enabled, we need to check the host state.
-    if srcdata.selinux {
```

Comment:
The re-exec is failing for some reason, I skipped it and things seems to work alright (maybe because I `sudo setenforce permissive`)

Reply:
update: it works even without setenforce

Reply:
Failing with what error message?

Reply:
```
ERROR Installing to filesystem: Ensuring selinux install_t type: Ensuring selinux install_t type: fsetxattr(security.selinux): Operation not supported (os error 95)
```

But I'm now starting to think this just might be due to my removal of `setup_tmp_mounts`

Reply:
Temporarily working around the `setup_tmp_mounts` error mentioned in the other comment below using `-v /tmp:/tmp -v /var/tmp:/var/tmp` 

But now the re-exec fails with a new error

```
ERROR Installing to filesystem: Ensuring selinux install_t type: Failed to enter install_t (running as unconfined_u:unconfined_r:spc_t:s0) - use BOOTC_SETENFORCE0_FALLBACK=1 to override
```

Reply:
Weirdly the error seems to disappear when `-v /tmp:/tmp` is removed. Confirming that this is true

Reply:
Yep that is indeed the case.

To summarize:

- No tmp mounts at all leads to `ERROR Installing to filesystem: Linking tmp mounts to host: Exchanging /var/tmp <=> /var/tmp.tmp: Invalid cross-device link (os error 18)`

- Both `/tmp/` and `/var/tmp` mounts leads to `ERROR Installing to filesystem: Ensuring selinux install_t type: Failed to enter install_t (running as unconfined_u:unconfined_r:spc_t:s0) - use BOOTC_SETENFORCE0_FALLBACK=1 to override`

- Only `/var/tmp` mount works